src/Simplex/Chat/Migrations/M20230621_chat_item_moderations.hs

```diff
@@ -0,0 +1,41 @@
+{-# LANGUAGE QuasiQuotes #-}
+
+module Simplex.Chat.Migrations.M20230621_chat_item_moderations where
+
+import Database.SQLite.Simple (Query)
+import Database.SQLite.Simple.QQ (sql)
+
+-- moderations that could not be applied - for messages that haven't been received at the time of moderation
+m20230621_chat_item_moderations :: Query
+m20230621_chat_item_moderations =
+  [sql|
+CREATE TABLE chat_item_moderations (
+  chat_item_moderation_id INTEGER PRIMARY KEY,
+  group_id INTEGER NOT NULL REFERENCES groups ON DELETE CASCADE,
+  moderator_member_id INTEGER NOT NULL REFERENCES group_members ON DELETE CASCADE,
+  item_member_id BLOB NOT NULL,
```

Comment:
why this column is needed? It should be available from group_members table

Reply:
or is it another member, the one who wrote the message, and we don't want to resolve it to DB ID at the time of the event - is it the same approach for reactions?

Reply:
yes